app/code/core/Mage/Catalog/Model/Convert/Parser/Product.php

```diff
@@ -507,7 +507,7 @@ public function unparse()
 
             $baseRowData = array(
                 'store'     => $row['store'],
-                'website'   => $row['website'],
+                'website'   => $row['websites'],
```

Comment:
Good find!

One question though, does the $baseRowData key need to change as well?